test/plugins/wasi_llm/wasi_llm.cpp

```diff
@@ -0,0 +1,199 @@
+      ModelTrain->getHostFunc());
+
+  std::array<WasmEdge::ValVariant, 1> Errno = {UINT32_C(0)};
+
+  std::string CheckPointString = "./wasi_llm/gpt2_124M.bin";
+  std::vector<char> CheckPointPath(CheckPointString.begin(),
+                                   CheckPointString.end());
+  uint32_t CheckPointPathPtr = UINT32_C(0);
+  writeBinaries<char>(MemInst, CheckPointPath, CheckPointPathPtr);
+
+  std::string TrainDataString = "./wasi_llm/tiny_shakespeare_train.bin";
+  std::vector<char> TrainDataPath(TrainDataString.begin(),
+                                  TrainDataString.end());
+  uint32_t TrainDataPathPtr = CheckPointPathPtr + CheckPointPath.size();
+  writeBinaries<char>(MemInst, TrainDataPath, TrainDataPathPtr);
+
+  std::string ValDataString = "./wasi_llm/tiny_shakespeare_val.bin";
+  std::vector<char> ValDataPath(ValDataString.begin(), ValDataString.end());
+  uint32_t ValDataPathPtr = TrainDataPathPtr + TrainDataPath.size();
+  writeBinaries<char>(MemInst, ValDataPath, ValDataPathPtr);
+
+  std::string TokenizerBin = "./wasi_llm/gpt2_tokenizer.bin";
+  std::vector<char> TokenizerBinPath(TokenizerBin.begin(), TokenizerBin.end());
+  uint32_t TokenizerBinPtr = ValDataPathPtr + ValDataPath.size();
+  writeBinaries<char>(MemInst, TokenizerBinPath, TokenizerBinPtr);
+
+  uint32_t ModelIdPtr = UINT32_C(0);
+  uint32_t ModelId = UINT32_C(0);
+  uint32_t TrainDataLoaderIdPtr = UINT32_C(0);
+  uint32_t TrainDataLoaderId = UINT32_C(0);
+  uint32_t ValDataLoaderIdPtr = UINT32_C(0);
+  uint32_t ValDataLoaderId = UINT32_C(0);
+  uint32_t TokenizerIdPtr = UINT32_C(0);
+  uint32_t TokenizerId = UINT32_C(0);
+
+  {
+    EXPECT_TRUE(HostFuncModelCreate.run(
+        CallFrame,
+        std::initializer_list<WasmEdge::ValVariant>{
+            CheckPointPathPtr, static_cast<uint32_t>(CheckPointPath.size()),
+            ModelIdPtr},
+        Errno));
+    EXPECT_EQ(Errno[0].get<int32_t>(), static_cast<uint32_t>(ErrNo::Success));
+    ModelId = *MemInst.getPointer<uint32_t *>(ModelIdPtr);
+    EXPECT_EQ(ModelId, 0);
+  }
+
+  {
+    EXPECT_TRUE(HostFuncDataLoadereCreate.run(
+        CallFrame,
+        std::initializer_list<WasmEdge::ValVariant>{
+            TrainDataPathPtr, static_cast<uint32_t>(TrainDataPath.size()),
+            /*B=*/4,
+            /*T=*/64,
+            /*ProcessRank=*/0,
+            /*NumProcesses*/ 1,
+            /*ShouldShuffle*/ 1, TrainDataLoaderIdPtr},
+        Errno));
+    EXPECT_EQ(Errno[0].get<int32_t>(), static_cast<uint32_t>(ErrNo::Success));
+    TrainDataLoaderId = *MemInst.getPointer<uint32_t *>(TrainDataLoaderIdPtr);
+    EXPECT_EQ(TrainDataLoaderId, 0);
+  }
+
+  {
+    EXPECT_TRUE(HostFuncDataLoadereCreate.run(
+        CallFrame,
+        std::initializer_list<WasmEdge::ValVariant>{
+            ValDataPathPtr, static_cast<uint32_t>(ValDataPath.size()),
+            /*B=*/4,
+            /*T=*/64,
+            /*ProcessRank=*/0,
```

Comment:
May I ask which style you want? I found that some of the comments have a space after it, some are not.

Reply:
i dont do this dirty work myself, but let clang-format take care of it.

Reply:
i removed the `=`, now the style should be consistent.